build.gradle

```diff
@@ -130,7 +131,7 @@ subprojects {
                 buildArgs << project.extraBuildArgs
 
             doFirst {
-                println buildArgs
+                logger.info buildArgs
```

Comment:
You can not `logger.info` a map.

Also, I think this was just for debugging, so it can be removed

Reply:
Removed